cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -4,7 +4,7 @@
     UnBondPeriod = 40
     MinStepValue = "100000000000000000000"
     AuctionEnableNonce = 100000000000
-    StakeEnableNonce = 0
```

Comment:
Restore all the modified flags in this file

Reply:
false